src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -203,7 +194,7 @@ public InternalPersonnelController()
 
             return new ApiCollection<ApiInternalPersonnelPerson>(returnModel);
         }
-
+        [MapToApiVersion("1.0")]
```

Comment:
Why is this needed?

Reply:
I Tried removing, but then a lot of swaggers dissapear. 
I have also compared pr vs CI environemnt and there is a lot of differences 
![image](https://github.com/equinor/fusion-app-resources/assets/3900367/7666d027-d904-4c06-8d5a-40116b0015e3)

Reply:
It appeasr the ones getting Obsolete is correct, but i am not able to not use     [MapToApiVersion("1.0")] on the InternalPerosnell endpoints that are supposed to b V1